bfx-pay-woocommerce.php

```diff
@@ -19,14 +19,45 @@
     return;
 }
 
+/**
+ * Register menu elements.
+ */
+class BFX_Admin_Menu
+{
+    public function __construct()
+    {
+        add_filter('plugin_action_links_' . plugin_basename( __FILE__), array($this, 'settings_link'), 10);
+    }
+
+    public function settings_link($links): array
+    {
+        $custom['settings'] = sprintf(
+            '<a href="%s" aria-label="%s">%s</a>',
+            esc_url(
+                add_query_arg(
+                    array(
+                        'page' => 'wc-settings&tab=checkout&section=bfx_payment',
+                    ),
+                    admin_url('admin.php')
+                )
+            ),
+            esc_attr__('Go to BFX Settings page', 'bfx-pay-woocommerce'),
+            esc_html__('Settings', 'bfx-pay-woocommerce')
+        );
+
+        return array_merge($custom, (array)$links);
+    }
+}
+
+new BFX_Admin_Menu();
+
 add_action('plugins_loaded', 'bfx_pay_woocommerce_init', 11);
-add_action('woocommerce_after_add_to_cart_form', 'bfx_pay_buy_checkout_on_archive');
```

Comment:
please restore old names as these were suggested by wp review team, we should use bfx_pay_ prefix